src/modules/Invoice/html_admin/mod_invoice_settings.html.twig

```diff
@@ -85,6 +85,12 @@
                         <input class="form-control" type="text" name="invoice_starting_number" value="{{ params.invoice_starting_number }}">
                     </div>
                 </div>
+                <div class="mb-3 row">
+                    <label class="col-md-3 col-form-label">{{ 'Invoice number zero padding width'|trans }}</label>
```

Comment:
I think "Invoice number zero padding width" is a bit wordy and might not be the most clear to some users.

I'd probably change it to say "Invoice number padding length" and add a short description under it giving a bit more detail on what it actually does

Reply:
Resolved with https://github.com/devadattas/FOSSBilling/commit/0bf8c921c9b174431ca305459a3776532af01102